HHS/Head-Start-TTADP
# [TTAHUB-3244] Show session status on event page

## Description of change

Add session status to event page, as depicted on [mockup](https://www.figma.com/design/IuIfn1FMTMTYS40XNf3sLw/View-session-status-on-view-event?node-id=1-2158&t=C1j4SLIJSEMj2jBc-1 )

## How to test
Easiest to view on a dataset with lots of events; view some and make sure the sessions have statuses

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3244


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
